tests/test_wheel.py

```diff
@@ -161,6 +161,13 @@ def test_local_lib(venv, wheel_link_against_local_lib):
     assert int(output) == 3
 
 
+@pytest.mark.skipif(MESON_VERSION < (0, 64, 0), reason='Meson version too old')
```

Comment:
What does break with older Meson versions?

Reply:
```
../meson.build:7:0: ERROR: python.find_installation got unknown keyword arguments "pure"
```

Reply:
Oh, this. We work around it in other test packages. I think we should either bump the minimum required Meson version or work around it here too passing the `pure` argument to the individual functions in the `python` module. Meson 0.64 was released November 6th 2022, thus almost 2 years ago. Maybe we can update the minimum required version to 0.64 without feeling bad 😃

Reply:
Yes, I'd prefer to bump to 0.64 rather than continue to use less idiomatic code. Especially since I plan to link to this test package from the docs as a complete and working example.

Reply:
Upgrade to 0.64.0 minimum version was done, so I dropped this line - resolving.